As vi is compiled with only FEAT_SMALL, it does not support many 
options which users may have in their vimrc.

--- origsrc/vim73/src/os_unix.h	2013-06-09 06:45:33.504200800 -0500
+++ src/vim73/src/os_unix.h	2013-06-09 07:03:37.315191300 -0500
@@ -241,7 +241,11 @@ typedef struct dsc$descriptor   DESC;
  * Unix system-dependent file names
  */
 #ifndef SYS_VIMRC_FILE
-# define SYS_VIMRC_FILE "$VIM/vimrc"
+# ifdef FEAT_NORMAL
+# define SYS_VIMRC_FILE "/etc/vimrc"
+# else
+# define SYS_VIMRC_FILE "/etc/virc"
+# endif
 #endif
 #ifndef SYS_GVIMRC_FILE
 # define SYS_GVIMRC_FILE "$VIM/gvimrc"
@@ -290,7 +294,11 @@ typedef struct dsc$descriptor   DESC;
 # ifdef VMS
 # define USR_VIMRC_FILE  "sys$login:.vimrc"
 # else
+#  ifdef FEAT_NORMAL
 #  define USR_VIMRC_FILE "$HOME/.vimrc"
+#  else
+#  define USR_VIMRC_FILE "$HOME/.virc"
+#  endif
 # endif
 #endif
 
@@ -340,7 +348,11 @@ typedef struct dsc$descriptor   DESC;
 #endif
 
 #ifndef VIMRC_FILE
+# ifdef FEAT_NORMAL
 # define VIMRC_FILE	".vimrc"
+# else
+# define VIMRC_FILE	".virc"
+# endif
 #endif
 
 #ifdef FEAT_GUI
